### DISCUSSION TITLE
Are plugins placed in ${ZSH_CUSTOM: folder updated?

### DISCUSSION THREAD

**Discussion by jkemp814** [ohmyzsh/ohmyzsh#11486] [Help] [answered]
I would like to know whether the zsh-users plugins update with the native plugins? 

I notice the antigen plugin manager hasn't been updated in years. When I add the zsh-users plugins via 'git clone' like zsh-completions, they are put in the ${ZSH_CUSTOM folder. So I've been rebasing each one.

**Comment by carlosala** [comment, marked as answer]
There's this pull request ongoing to implement that, we are still working on it!
https://github.com/ohmyzsh/ohmyzsh/pull/10772
